<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUserWindow</class>
 <widget class="QDialog" name="MainUserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User page</string>
  </property>
  <widget class="QLabel" name="ServerStateLabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Server is connected</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="WelcomeLabel">
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Segoe UI&quot;;
color: rgb(255, 170, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#0000ff;&quot;&gt;Welcome &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="WelcomeLabel_2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 170, 255);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff0004;&quot;&gt;Role: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="PB_GetAccountNumber">
      <property name="text">
       <string>Get Account Number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_ViewAccountBalance">
      <property name="text">
       <string>View Account Balance</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_ViewTransactionHistory">
      <property name="text">
       <string>View Transaction History</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_MakeTransaction">
      <property name="text">
       <string>Make Transaction</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_TransferAmount">
      <property name="text">
       <string>Transfer Amount</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>561</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="GetAccountNumber">
    <widget class="QLabel" name="AccNumLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>541</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Your account number is&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ViewAccountBalance">
    <widget class="QLabel" name="BalanceLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>541</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Your account balance=&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ViewTransactionHistory">
    <widget class="QTableView" name="TV_ViewTransactionHistory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>211</height>
      </rect>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="MakeTransaction">
    <widget class="QLabel" name="TransactionBalanceLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Your Balance = </string>
     </property>
    </widget>
    <widget class="QLabel" name="Status">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>481</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Transaction amount: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_TransactionAmount">
          <property name="inputMethodHints">
           <set>Qt::ImhDialableCharactersOnly|Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Please enter your signature: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_TransactionSignature_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="PB_Deposit">
          <property name="text">
           <string>Deposit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_Withdraw">
          <property name="text">
           <string>Withdraw</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="TransferAmount">
    <widget class="QLabel" name="TransferBalanceLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Your balance= </string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>521</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Please enter the account ID: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_TransferAccID"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Please enter the amount to transfer: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_TransferAmount"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Please enter your signature: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_TransferSignature"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Transfer">
        <property name="text">
         <string>Transfer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="EmptyPage"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
